test/mz-e2e/mzcompose.py

```diff
@@ -0,0 +1,254 @@
+        except UIError:
+            # Work around https://github.com/MaterializeInc/materialize/issues/17219
+            pass
+
+        time.sleep(10)
+
+        assert "materialize.cloud" in cloud_hostname(c)
+        wait_for_cloud(c)
+
+        # if args.version_check:
+        #     version_check(c)
+
+        # Test - `mz app-password`
+        # Assert `mz app-password create`
+        new_app_password_name = "Materialize CLI (mz) - Nightlies"
+        output = c.run(
+            "mz", "app-password", "create", new_app_password_name, capture=True
+        )
+        assert "mzp_" in output.stdout
+
+        # Assert `mz app-password list`
+        output = c.run("mz", "app-password", "list", capture=True)
+        assert new_app_password_name in output.stdout
+
+        # // Test - `mz secrets`
+        secret = "decode('c2VjcmV0Cg==', 'base64')"
+        output = c.run(
+            "mz", "secret", "create", "CI_SECRET", stdin=secret, capture=True
+        )
+        assert output.returncode == 0
+
+        output = c.run("mz", "sql", "--", "-q", "-c", "SHOW SECRETS;", capture=True)
+        assert "ci_secret" in output.stdout
+
+        # Assert using force
+        output = c.run(
+            "mz", "secret", "create", "CI_SECRET", "force", stdin=secret, capture=True
+        )
+        assert output.returncode == 0
+
+        # Test - `mz user`
+        user_email = "mz_ci_e2e_test_nightlies@materialize.com"
+
+        # Try to remove the username if it exist before trying to create one.
+        try:
+            output = c.run("mz", "user", "remove", user_email, capture_stderr=False)
+            print("Warning: Email was present.")
+        except:
+            # It is ok if the command fails.
+            pass
+
+        output = c.run("mz", "user", "create", user_email, "MZ_CI", capture=True)
+        assert output.returncode == 0
+
+        output = c.run("mz", "user", "list", capture=True)
+        assert output.returncode == 0
+        assert user_email in output.stdout
+
+        output = c.run("mz", "user", "remove", user_email, capture=True)
+        assert output.returncode == 0
+
+        output = c.run(
+            "mz",
+            "user",
+            "list",
+            "--format",
+            "csv",
+            capture=True,
+        )
+        # assert username is not in output.stdout
+        user_list = csv.DictReader(output.stdout.split("\n"))
+        for user in user_list:
+            assert user["email"] != user_email
+
+        # Test - `mz region list`
+        # Enable, disable and show are already tested.
+        output = c.run("mz", "region", "list", "--format", "json", capture=True)
+        regions = json.loads(output.stdout)
+        us_region = regions[0]
+        if us_region["region"] != "aws/us-east-1":
+            us_region = regions[1]
+        assert "enabled" == us_region["status"]
+
+        # Look for the '.psqlrc' file in the home dir.
```

Comment:
Which command is supposed to create this file? Consider making sure the file does not exist to start with, before running the command.